red_queen/games/applications/run_bv.py

```diff
@@ -58,19 +78,22 @@ def build_bv_circuit(secret_string, mid_circuit_measure=False):
 @pytest.mark.parametrize("optimization_level", [0, 1, 2, 3])
 @pytest.mark.parametrize("backend", backends)
 @pytest.mark.parametrize("method", ["normal", "mid-circuit measurement"])
-def bench_qiskit_bv(benchmark, optimization_level, backend, method):
+@pytest.mark.parametrize("num_qubits", sweep_list)
+def bench_qiskit_bv(benchmark, optimization_level, backend, method, num_qubits):
     shots = 65536
+    SECRET_STRING = str(bin(random.getrandbits(num_qubits - 1))[2:].zfill(num_qubits-1))
     expected_counts = {SECRET_STRING: shots}
+
     if method == "normal":
         benchmark.name = "Bernstein Vazirani"
-        circ = QuantumCircuit.from_qasm_file(os.path.join(QASM_DIR, "bv.qasm"))
+        circ = build_bv_circuit(SECRET_STRING)
     else:
         benchmark.name = "Bernstein Vazirani (mid-circuit measurement)"
-        circ = QuantumCircuit.from_qasm_file(os.path.join(QASM_DIR, "bv_mcm.qasm"))
+        circ = build_bv_circuit(SECRET_STRING, True)
     benchmark.algorithm = f"Optimization level: {optimization_level} on {backend.name()}"
     run_qiskit_circuit(benchmark, circ, backend, optimization_level, shots, expected_counts)
 
 
 if __name__ == "__main__":
-    build_bv_circuit(SECRET_STRING).qasm(filename=os.path.join(QASM_DIR, "bv.qasm"))
-    build_bv_circuit(SECRET_STRING, True).qasm(filename=os.path.join(QASM_DIR, "bv_mcm.qasm"))
+    build_bv_circuit(default_secret_string).qasm(filename=os.path.join(QASM_DIR, "bv.qasm"))
+    build_bv_circuit(default_secret_string, True).qasm(filename=os.path.join(QASM_DIR, "bv_mcm.qasm"))
```

Comment:
Since we're changing the execution model of these benchmarks to no longer depend on hard coded qasm files we can delete this block of code now as we don't need to regenerate qasm files. Similarly we can remove the `bv*.qasm` files at the same time because nothing will use them anymore.